src/elm/Page/Register.elm

```diff
@@ -193,43 +193,45 @@ view guest model =
             [ onSubmit ValidateForm
             ]
             [ div [ class "min-w-full md:min-w-0 md:mx-auto px-4" ]
-                [ p [ class "text-white" ]
-                    [ text "Step 1 of 2 / Basic informations" ]
-                , p
-                    [ class "text-grey" ]
-                    [ text_ "register.form.title" ]
+                [ p [ class "py-4 mb-4 text-grey text-body border-grey-500 border-b border-dotted " ]
+                    [ text "Step 1 of 2 / "
+                    , strong [ class "text-black" ] [ text "Basic Information" ]
+                    ]
                 , viewServerErrors model.problems
-                , viewField shared
+                , viewField
+                    shared
+                    model
                     (Field
                         "register.form.name"
                         isDisabled
-                        "name"
+                        model.form.username
```

Comment:
It's convenient to use translation suffix as an `id` in HTML. Here I pass current value of the field to use it in view rendering.